apiservers/engine/backends/image.go

```diff
@@ -49,7 +64,71 @@ func (i *Image) ImageHistory(imageName string) ([]*types.ImageHistory, error) {
 }
 
 func (i *Image) Images(filterArgs string, filter string, all bool) ([]*types.Image, error) {
-	return nil, fmt.Errorf("%s does not implement image.Images", i.ProductName)
+
+	result := []*types.Image{}
+
+	host, err := os.Hostname()
+	if err != nil {
+		return result,
+			derr.NewErrorWithStatusCode(fmt.Errorf("image.Images got unexpected error getting hostname"),
+				http.StatusInternalServerError)
+	}
+
+	params := &storage.ListImagesParams{
+		StoreName: host,
+	}
```

Comment:
I would be down for this with a sane default as it would mean that we don't have to get the hostname in every handler.